lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -403,6 +515,23 @@ export default {
 		};
 	},
 	computed: {
+		classList() {
+			return {
+				'-default': this.type === RICH_LIST_ITEM_TYPE.DEFAULT,
+				'-flat': this.type === RICH_LIST_ITEM_TYPE.FLAT,
+				'-loading': this.state === RICH_LIST_ITEM_STATE.LOADING,
+				'-dimmed': this.isDimmed,
```

Comment:
I cannot comment style, so I will put it here. Shouldn't the opacity has value 0.5? I see such on figma 
![Zrzut ekranu 2024-03-13 o 20 39 54](https://github.com/bethinkpl/design-system/assets/11304259/28c92b39-b7bf-41d7-8118-e07e93de3626)

Reply:
But this is loading state, not dimmed. https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS-Components?type=design&node-id=8507-129792&mode=dev

Reply:
Ach. I could not find it first time, but now I see it in figma